app/helpers/application_helper.rb

```diff
@@ -41,7 +41,11 @@ def t_lang(key, options = {})
   end
 
   def t_fallback(key, options = {})
-    translation = I18n.t(key, options, locale: I18n.locale, fallback: false, default: "fallback")
```

Comment:
I think an alternative to the code you've got would be to use `**options` instead.

```rb
I18n.t(key, locale: I18n.locale, fallback: false, default: "fallback", **options)
```

I don't think we need to do the `rescue` with that either.